.env.development

```diff
@@ -13,6 +13,6 @@ AWS_SECRET_ACCESS_KEY_BOOKBAZAR=
 AWS_BUCKET_NAME_BOOKBAZAR=
 AWS_REGION_BOOKBAZAR=
 
-ALGOLIA_APP_ID=
-ALGOLIA_API_KEY=
-ALGOLIA_INDEX_NAME=
\ No newline at end of file
+NEXT_PUBLIC_ALGOLIA_APP_ID=
+NEXT_PUBLIC_ALGOLIA_API_KEY=
+NEXT_PUBLIC_ALGOLIA_INDEX_NAME=
```

Comment:
Don't we still need the old write-access API key as well for the CRON job? 

Please search the project for, `ALGOLIA_API_KEY` (etc), there are a few places where these need to be updated. Also just a reminder to make sure to update these in our [repository action secrets](https://github.com/calebmech/Book-Bazar/settings/secrets/actions) in addition to Vercel.

Reply:
Don't belive I have access to the settings
<img width="804" alt="Screen Shot 2022-02-07 at 9 16 10 PM" src="https://user-images.githubusercontent.com/40670885/152905101-05123af5-824e-44cb-b581-08b4ba961fbf.png">
?

Reply:
Ah, yeah I can take care of that after your changes are finished then

Reply:
@alkoasma This issue still needs to be addressed (it's not just updating secrets on Vercel and GitHub).

Reply:
Thanks for reminding me of that. I made the required changes.

Reply:
Actually, you can confirm, but based on my understanding of the code updates, I don't believe you need to change anything related to the secrets.